tests/infra_test.go

```diff
@@ -688,20 +690,12 @@ var _ = Describe("[Serial]Infrastructure", func() {
 					} else {
 						resp.Body.Close()
 					}
-					errors <- err
+					errorsChan <- err
 				}(ix)
```

Comment:
I would move the error treatment (`validatedHTTPResponses`) to here. So if we get a timeout error we can break the loop.